central/sensor/service/connection/sensorevents.go

```diff
@@ -77,6 +81,11 @@ func (s *sensorEventHandler) addMultiplexed(ctx context.Context, msg *central.Ms
 	switch event.Resource.(type) {
 	case *central.SensorEvent_Synced:
 		// Call the reconcile functions
+		log.Info("Receiving reconciliation event")
+		if s.reconciliationMap.IsClosed() {
+			log.Info("Ignoring SYNC since reconciliationMap is already closed")
```

Comment:
Add cluster information to this log because otherwise we could get 100 of these and not be able to trace anything. Same for the receiving reconciliation event above

Reply:
Done